test/partitioning.jl

```diff
@@ -196,7 +196,31 @@
   end
 
   @testset "SpatialPredicatePartitioner" begin
-    # TODO
+    g = RegularGrid{Float64}(10,10)
+    R = 5.0
+    SPred1(x,y) = norm(x-y) < R
```

Comment:
This predicate is a small one-line function, please use a lambda to construct the partitioner:

```Julia
p = partition(g, SpatialPredicatePartitioner((x, y) -> norm(x-y) < 5.0)
```

And then take the subsets directly in one line:

```julia
s = subsets(partition(g, p))
```

Reply:
you meant to say `p = SpatialPredicatePartitioner((x, y) -> norm(x-y) < 5.0)`, and then `s = subsets(partition(g, p))`, right?

Reply:
Yes, marking as solved.